packages/server-wallet/src/handlers/join-channel.ts

```diff
@@ -43,8 +43,8 @@ function ensureLatestStateIsPrefundSetup(cs: ChannelState): StepResult {
   return left(new JoinChannelError(JoinChannelError.reasons.invalidTurnNum));
 }
 
-function latest(cs: ChannelState): StateVariables {
-  return cs.latest;
+function nextState(cs: ChannelState): StateVariables {
```

Comment:
Maybe this name is a bit more clear?
```suggestion
function myPrefundState(cs: ChannelState): StateVariables {
```